cmake/offline.cmake

```diff
@@ -25,3 +25,18 @@ target_add_mason_package(mbgl-offline PRIVATE boost_libprogram_options)
 mbgl_platform_offline()
 
 create_source_groups(mbgl-offline)
+
+xcode_create_scheme(
+    TARGET mbgl-offline
+    OPTIONAL_ARGS
+        "--style=file.json"
+        "--north=37.2"
+        "--west=-122.8"
+        "--south=38.1"
+        "--east=-121.7"
+        "--minZoom=0.0"
+        "--maxZoom=15.0"
+        "--pixelRatio=1.0"
+        "--token=pk.D4Ft0FqCvDiEqomHkG0z1c9iszRoWC2KgyTkfJeC.I5ultYarQMQRf4lLKUoU"
```

Comment:
It has never been a real token; it's a random string I generated. Note that these options are disabled by default, they're just there to show the user what's possible.